sources/api/openapi.yaml

```diff
@@ -270,7 +270,30 @@ paths:
           description: "Invalid body"
         500:
           description: "Server error"
-
+  /settings/keypair/:
+    patch:
+      summary: "Update settings which come in key value form"
+      operationId: "set_key_pair_settings"
+      parameters:
+        - in: query
+          name: tx
+          description: "Transaction in which to update settings; defaults to user 'default' transaction"
+          schema:
+            type: string
+          required: false
+      requestBody:
+        required: true
+        content:
+          text/plain:
+            schema:
+              type: string
```

Comment:
It this right? Isn't it a JSON of key/value pairs?

Reply:
Earlier we would deserialize directly into Settings Model and just serialize that to send over the network. Now we changed the Model to be a simple json string for /settings and a 

```
{
"request_payload": Vec<String>
}
```
for the /settings/keypair type.

But youre right, I should update this with a new Schema and it should be JSON.